### PR TITLE
[Legacy] Corrected aqi label

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)
- Corrected AQI label

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):
- [x] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* git pull origin staging
* git checkout hotfix-AQI-label
* npm run stage

pm2 5

### CODE REVIEW

File: platform/.gitignore [hunk 1]
```diff
@@ -71,6 +71,7 @@ typings/
 # dotenv environment variables file
 .env
 .env.test
+.env.local
```
Comment (Codebmk) [inline platform/.gitignore:74]: I was working on the platform repo and added a new env file. This simply ignores the env file so it's not added to the repo

**Review by 123MwanjeMike — APPROVED** [review]
LGTM

**Comment by FaithDaka** [comment]
Thanks @Codebmk 
A few thoughts: the PM label should follow its convention i.e PM<sub>2.5</sub> vs PM2.5, which also means the units should be shown as well (µg/m<sup>3</sup>). Might need to reduce the label font-size accordingly to maintain the design.

**Comment by Codebmk** [comment]
> Thanks @Codebmk A few thoughts: the PM label should follow its convention i.e PM2.5 vs PM2.5, which also means the units should be shown as well (µg/m3). Might need to reduce the label font-size accordingly to maintain the design.

Thanks @FaithDaka for catching that

**Comment by Codebmk** [comment]
> Thanks @Codebmk A few thoughts: the PM label should follow its convention i.e PM2.5 vs PM2.5, which also means the units should be shown as well (µg/m3). Might need to reduce the label font-size accordingly to maintain the design.

These changes are ready @FaithDaka

**Comment by FaithDaka** [comment]
> > Thanks @Codebmk A few thoughts: the PM label should follow its convention i.e PM2.5 vs PM2.5, which also means the units should be shown as well (µg/m3). Might need to reduce the label font-size accordingly to maintain the design.
> 
> These changes are ready @FaithDaka

The units are put next to the value @Codebmk since the value is the measurement

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**